src/algebra/star/basic.lean

```diff
@@ -203,6 +203,13 @@ variables (R)
 
 variables {R}
 
+@[simp]
+lemma star_eq_zero [add_monoid R] [star_add_monoid R] {x : R} : star x = 0 ↔ x = 0 :=
+by { nth_rewrite 0 ← star_zero, exact ⟨λ h, star_injective h, λ h, star_injective (by simpa)⟩ }
```

Comment:
I think this should have a trivial proof as something like `star_add_equiv.map_eq_zero_iff`

Reply:
Indeed, exactly as you say!